Automattic/jetpack
# Subscriptions block updates

Update the email subscription block to significantly improve the settings for customizing the display and style of the form. 

Before                                   |  After
:-------------------------:|:-------------------------:
<img width="1214" alt="before" src="https://user-images.githubusercontent.com/1464705/78055519-854cff00-7338-11ea-8115-62144996c291.png">  | <img width="1204" alt="after" src="https://user-images.githubusercontent.com/1464705/78055517-841bd200-7338-11ea-891c-3b61991843ae.png">

Fixes #15076

#### Testing instructions:
* Insert a subscriptions form block on your site and try all of the new settings in the sidebar for customization.
* Save the post and confirm the settings and design updates pass through to the front end of the site.
* Add the subscription form widget to a widget area and confirm this still renders and works correctly.

To test old blocks:
* Add the block in master
* Check out this branch
* Check that the block looks ok in the editor and in the frontend

#### Proposed changelog entry for your changes:

* Will update this later.
